server/src/main/java/org/apache/druid/query/lookup/LookupReferencesManager.java

```diff
@@ -168,7 +168,7 @@ public void start()
                 while (!Thread.interrupted() && lifecycleLock.awaitStarted(1, TimeUnit.MILLISECONDS)) {
                   try {
                     handlePendingNotices();
-                    LockSupport.parkNanos(LookupReferencesManager.this, TimeUnit.MINUTES.toNanos(1));
+                    LockSupport.parkNanos(LookupReferencesManager.this, TimeUnit.SECONDS.toNanos(1));
```

Comment:
Why is this being changed?

Reply:
Here I tried to increase the frequency, from per minute to per second, to poll the status of lookup loading.

Reply:
I feel that per second is more aggressive given that lookup updates may not happen that frequently in most cases and per minute is more reasonable since it is consistent with the default run period of the coordinator.